components/gc-follow-us/_base.scss

```
@@ -310,7 +314,7 @@ $social-media-icons-size: 38px;
 	}
 
 	&.twitter {
-		@extend %social-media-icons-logo-twitter;
+		@extend %social-media-icons-logo-x;
 	}
 
```

Should we add a new style exclusively for the social media X? I am thinking a css class like `social-media-x` or something similar.

```suggestion

	&.social-media-x {
		@extend %social-media-icons-logo-twitter;
		@extend %social-media-icons-logo-x;
	}

```

---

Added ".x-social"